lms/templates/bookmark_button.html

```diff
@@ -0,0 +1,11 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%page args="is_bookmarked, bookmark_id" />
+
+<div class="bookmark-button-wrapper">
+  <button class="btn bookmark-button ${"bookmarked" if is_bookmarked else "un-bookmarked"}"
+    aria-pressed="${"true" if is_bookmarked else "false"}"
+    data-bookmark-id="${bookmark_id}">
+     <span class="sr bookmark-sr">${_("Click to remove bookmark from this unit") if is_bookmarked else _("Click to bookmark this unit")}</span>
```

Comment:
@srpearce Can you confirm if this will be fine from the point of view of translation?